lava_test_plans/include/fastboot.jinja2

```diff
@@ -22,7 +22,7 @@
 {% set download_postprocess_required = download_postprocess_required|default(true) %}
 
 {# partitions for android jobs #}
-{% set partition_boot_a = partition_boot_a|default(false) %}
+{# boot_a partition would be set via boot_label #}
```

Comment:
I assumed this should have been squashed into 220557ed56c2 ("include: fastboot: Add support for android partitions")

Reply:
updated